paddle/operators/cross_entropy_op.cc

```diff
@@ -17,59 +17,71 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
-class OnehotCrossEntropyOp : public framework::OperatorWithKernel {
+class CrossEntropyOp : public framework::OperatorWithKernel {
```

Comment:
@luotao1 changes CrossEntropyOp -> OnehotCrossEntropyOp. Please use the new name.

Reply:
Since it supports not only one-hot cross-entropy but also soft-label cross-entropy, it would be better to use CrossEntropyOp instead of OnehotCrossEntropyOp.